metagraph/src/cli/annotate.cpp

```diff
@@ -358,8 +358,10 @@ void annotate_data(std::shared_ptr<graph::DeBruijnGraph> graph,
                     }
                 );
             } else {
-                logger->warn("No k-mer counts found at '{}'. Every input k-mer"
-                             " will have count 1.", counts_fname);
+
+                logger->warn("No k-mer counts found at '{}', "
+                             "will try to deduce counts from headers",
+                             counts_fname);
```

Comment:
```suggestion
```

Reply:
it should already print the message in the callback, right?

Reply:
I meant this: https://github.com/ratschlab/metagraph/pull/558/files#diff-a94f0a7cafe1cfcfe853836f6133f23cf8e8725270339d1127d7835213bca477R186-R188

And yeah, we shouldn't print it for every sequence, just once.